api/rds.proto

```diff
@@ -585,6 +588,22 @@ message RedirectAction {
   RedirectResponseCode response_code = 3 [(validate.rules).enum.defined_only = true];
 }
 
+message DirectResponseAction {
+  // Specifies the HTTP response status to be returned.
+  google.protobuf.UInt32Value status = 1 [(validate.rules).message.required = true];
+
+  // Specifies the human-readable status to be returned if
+  // the response is encoded as HTTP/1.x.
+  string reason = 2;
+
+  // Specifies the response content-type. This setting defaults to text/plain.
+  string content_type = 3;
+
+  // Specifies the content of the response body. If this setting is omitted,
+  // no body is included in the generated response.
+  string body = 4;
```

Comment:
I think this should be a DataSource, so that it can either be a file on disk, or provided inline.